### ISSUE TITLE
rstudio IDE freeze after entering ^ character in editor (french keyboard)

### ISSUE THREAD

**Issue by hansivers** [rstudio/rstudio#4795]
### System details

    RStudio Edition :  Desktop
    RStudio Version : 1.2.1335
    OS Version      :  Windows 7 PRO ENG
    R Version       :  3.6.0

### Steps to reproduce the problem
open Rstudio
create new r script
with french keyboard, enter ^ character (using the key at the right of the p letter on a QWERTY keyboard)
ide freeze 
I reproduced this bug on two different computers, with the same setup

**Comment by ronblum** [comment]
@hansivers Thank you for reporting the issue. I'm unable to reproduce it, though, using Windows 7 with language and keyboard set to French (France). 

1) What version of Windows 7 are you using? I'm on Windows 7 Professional Service Pack 1, build 7601 (version 6.1.7601). 
2) When I press <kbd>^</kbd>, the first time doesn't result in anything. The second time, <kbd>^</kbd> appears twice. This is true elsewhere, including Notepad and Google Chrome. Does this happen to you, too?

**Comment by hansivers** [comment]
Dear Ron,
1. Same Windows 7 build as you
2. I'm actually using the French (Canada) Keyboard. I see the same expected behavior as you (press twice to have the "hat" accent in French) in any other application, but RStudio IDE. 

Actually, I found that the bug is happening more cleanly when I switched from English to French keyboard :
1. Using Windows Language Bar, I've mapped the sequence "Ctrl + 1" to get English keyboard, and "Ctrl + 2" to get French (Canada) keyboard (quick way to change between language when programming) 
2. The freeze happened systematically when I pressed the "hat" key AFTER switching from English to French keyboard using the ctrl keys.
3. If I switched between keyboards using the mouse, there is no freeze when typing the hat key.

I've downgraded to RStudio 1.1.463 and everything working as expected (there is no interference between switching keyboard using Ctrl keys and the hat key)

Thanks for your help,

Hans
